src/Makefile.am

```diff
@@ -12,7 +12,7 @@ noinst_HEADERS = \
 	alert-fastlog.h \
 	alert-prelude.h \
 	alert-syslog.h \
-	app-layer-dcerpc-common.h \
```

Comment:
this looks suspicious

Reply:
Indeed, most likely an editing oops made during merge conflict resolution. Will fix